src/components/DailyChecklist.tsx

```diff
@@ -78,12 +106,11 @@ const Tasks = ({ day }: Props) => {
     );
   }
   return (
-    <div>
-      <button type="submit" onClick={handleClick}>
-        edit
-      </button>
-      {tasks.map((task) => (
-        <Checkbox key={uuidv4()} task={task} />
+    <div id="tasks">
+      {tasks.map((task, index) => (
+        <div key={uuidv4()} id="editing">
```

Comment:
I feel like the id "editing" doesn't really make sense in this context

Reply:
fdd48ba